src/tiled-map-resource.ts

```diff
@@ -42,6 +42,18 @@ export enum TiledMapFormat {
    JSON = 'JSON'
 }
 
+export interface TiledMapOptions {
+   /**
+    * Optionally override the map format, by default files ending in .tmx are treated as TMX format, otherise treated as JSON format
```

Comment:
```suggestion
    * By default files ending in .tmx are treated as TMX format, otherwise treated as JSON format
```

Shortened (option name is clear), fixed typo